tox.ini

```diff
@@ -186,4 +186,4 @@ extras =
     pyqt6
 commands =
     python -m napari --info
-    python -m napari
+    python -P -m napari
```

Comment:
what does this do?

Reply:
-P avoid importing from local directory by not putting local directory in PYTHONPATH

Reply:
🤯 TIL 🙏